komodo/snyk_reporting.py

```diff
@@ -41,14 +41,36 @@ def main() -> None:
 
 def parse_args(args: Dict[str, str]) -> argparse.Namespace:
     parser = argparse.ArgumentParser(
-        description="Test a release for security and license issues"
+        description="Test a release for security and license issues",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    )
+    parser.add_argument(
+        "--orgid", type=str, required=True, help="The Snyk organization ID"
+    )
+    parser.add_argument(
+        "--repo",
+        type=YamlFile(),
+        required=True,
+        help="A Komodo repository file, in YAML format.",
```

Comment:
small k in Komodo. Same comment for the instance below.

Reply:
I think most mentions are now uppercase so I think I'll leave this for now. I don't think there's any ambiguity.